docs/Developers/Review.rst

```diff
@@ -117,6 +117,17 @@ For the Reviewers
    behaviour well. E.g. for bear tests just testing for a good and bad file
    is **not** sufficient.
 
+.. note::
+
+    While reviewing pull request or patch for ``difficulty/low`` issue,
+    make sure that the patch solves the issue and doesn't create any
+    further issues.
+
+    You need to thoroughly review the code, i.e. understand the functionality
+    of the code, check whether it is efficient or not, and leave critical
+    comments. Otherwise, don't review!. We need human reviews to find the
+    problems which can't be found automatically.
```

Comment:
Maybe add some sentence that a good review takes time? Up to you

Reply:
does issue demands this?, although if you want I will add it definetely

Reply:
up to you :)

Reply:
Its simple statement for every reviewers either review efficiently or don't review